flict/flictlib/format/text_format.py

```diff
@@ -19,13 +19,11 @@ def __init__(self):
         self.col_size = 18
 
     def format_support_licenses(self, supported_licenses):
-        ret_str = ""
-        for item in supported_licenses:
-            ret_str += " " + str(item) + "\n"
-        return ret_str
+        ret = [f" {item}" for item in supported_licenses]
+        return "\n".join(ret)
```

Comment:
even simpler it would be `return " \n".join({str(item) for item in supported_licenses])`

Reply:
If I understood it correctly, the intend was to have a space in front of the name. " item\n" is the format.

Reply:
so it's `return "\n" + "\n".join(str(item) for item in supported_licenses])` anyway leave it like it is, I'm fine with that